docs/articles/documentation/using-testcafe/configuration-file.md

```diff
@@ -0,0 +1,764 @@
+---
+layout: docs
+title: Configuration File
+permalink: /documentation/using-testcafe/configuration-file.html
+---
+# Configuration File
+
+TestCafe uses the `.testcaferc.json` configuration file to store its settings.
+
+> Important! Settings you specify when you run TestCafe from the [command line](command-line-interface.md) and [programming interfaces](programming-interface/README.md) override settings from `.testcaferc.json`. TestCafe prints information about every overridden property in the console.
+
+Keep `.testcaferc.json` in a directory from which you run TestCafe. Most often, this is the project's root directory. TestCafe does not take into account configuration files located in other directories (for instance, project's subdirectories).
```

Comment:
in a directory -> in **the** directory

Reply:
Keep `.testcaferc.json` in the directory from which you run TestCafe. This is usually the project's root directory.

Reply:
I do not think it is necessary to add 'TestCafe does not take into account configuration files located in other directories (for instance, project's subdirectories)'.

Reply:
Well, many (or even most) libraries do search for configuration files in subdirectories and then merge them with the base directory's configuration file. We don't do this, but I think some people could assume that we do. That's why I thought it was worth mentioning.